src/index.ts

```diff
@@ -271,14 +271,23 @@ async function appBoot() {
 			await migrateFromWpNowFolder();
 		}
 
+		const userData = await loadUserData();
+
+		if ( ! userData.sentryUserId ) {
+			userData.sentryUserId = crypto.randomUUID();
```

Comment:
WDYT about `studioInstanceId`?

```suggestion
			userData.studioInstanceId = crypto.randomUUID();
```

Reply:
This is what I referred to in the PR description 😄

> Per @nightnei's suggestion, I considered naming the `sentryUserId` key in the user config differently to indicate that it could be used for other reporting purposes in the future. However, I concluded that this is hypothetical, and it's better to optimize for the scenario at hand by using an explicit name.

To elaborate, I think a name like `studioInstanceId` is more difficult to understand at a glance, and there's currently no plan to use it for other purposes than Sentry reporting.

To that end, I think it's better to cross that bridge when and if we get there.